module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -1428,15 +1440,15 @@ public function getHoldDefaultRequiredDate($patron, $holdInfo)
      * placeHold, minus the patron data.  May be used to limit the pickup options
      * or may be ignored.
      *
-     * @return string A location ID
+     * @return false|string      The default pickup location for the patron or false
```

Comment:
We should probably keep wording in the return hint to indicate that it's a location ID we're looking for when a string is returned. Also, once we have standard return value wording, should we update the driver spec and copy it into all the other drivers that implement this method?

Reply:
Ah, yes. I just copied it from another driver. Does "Default pickup location ID for the patron, or false for no default value" make sense for the wording?

Reply:
Works for me! And I thought this was a new feature, but apparently you're just leveraging an existing one. Sorry for the confusion. :-)

Reply:
No problem! It's existing functionality supported by several drivers, and it looks like our specs have this case covered, so no need to fix there. I was a bit worried for a minute! :)